language/move-core/types/src/gas_schedule.rs

```diff
@@ -151,127 +151,5 @@ pub const REFERENCE_SIZE: AbstractMemorySize<GasCarrier> = AbstractMemorySize(8)
 /// The size of a struct in bytes
 pub const STRUCT_SIZE: AbstractMemorySize<GasCarrier> = AbstractMemorySize(2);
 
-/// For V1 all accounts will be ~800 bytes
-pub const DEFAULT_ACCOUNT_SIZE: AbstractMemorySize<GasCarrier> = AbstractMemorySize(800);
-
-/// Any transaction over this size will be charged `INTRINSIC_GAS_PER_BYTE` per byte
-pub const LARGE_TRANSACTION_CUTOFF: AbstractMemorySize<GasCarrier> = AbstractMemorySize(600);
-
 /// For exists checks on data that doesn't exists this is the multiplier that is used.
 pub const MIN_EXISTS_DATA_SIZE: AbstractMemorySize<GasCarrier> = AbstractMemorySize(100);
-
-pub const MAX_TRANSACTION_SIZE_IN_BYTES: GasCarrier = 4096;
-
-#[derive(Clone, Debug, Serialize, PartialEq, Deserialize)]
-pub struct GasConstants {
```

Comment:
I saw the change elsewhere, but didn't quite grok, why the move?

Reply:
With the addition of the `trait GasMeter`, adapters are now likely to implement their own gas meters. The `GasStatus` carries a lot of legacies and is quite complicated TBH. I want to avoid giving clients the impression that we still have this official `GasMeter` implementation, at least before we have the time to clean it up a bit.